tools/gomod-local-update/README.md

```diff
@@ -0,0 +1,32 @@
+# gomod-local-update
```

Comment:
Should it be added that you can/should use the `gomods` tool to fix all `go.mod` files in a repo?

Also maybe note that it will only fix `go.mod` in cwd? (I think that's right).

Reply:
63638a7989320530b0ffbc10c56d6372e3add0a8